flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/AvroRowDataDeserializationSchema.java

```diff
@@ -86,6 +86,27 @@ public AvroRowDataDeserializationSchema(
                 typeInfo);
     }
 
+    /**
+     * Creates an Avro deserialization schema for the given logical type.
+     *
+     * @param rowType The logical type used to deserialize the data.
+     * @param typeInfo The TypeInformation to be used by {@link
+     *     AvroRowDataDeserializationSchema#getProducedType()}.
+     * @param encoding The serialization approach used to deserialize the data.
+     * @param legacyMapping Whether to use legacy mapping.
```

Comment:
the scope meaning of `legacyMapping` is too large for our case, what we want to express is only timestamp  types' legacy behavior, but this name looks like we have a legacy behavior for all types. How about `legacyTimestampMapping`?